crates/grafbase-sdk/src/types/directive.rs

```diff
@@ -58,6 +58,15 @@ impl<'a> FieldDefinitionDirective<'a> {
         minicbor_serde::from_slice(&self.0.arguments).map_err(Into::into)
     }
 
+    /// Deserialize the arguments of the directive using a `DeserializeSeed`.
+    pub fn deserialize_arguments_seed<T>(&self, seed: T) -> Result<T::Value, SdkError>
```

Comment:
Thinking about it I think you could have just used `arguments_seed` for consistency.

Reply:
Good point, will change next PR